### ISSUE TITLE
Rename devel to master branch

### ISSUE THREAD

**Issue by pabelanger** [ansible/network#5]
Hello,

for legacy reasons, we seem to create new projects by rm -rf master; git branch devel. However, our developer workflow doesn't actually fit the develop branch model historically used by developers.  We don't actually commit code to devel, then merge to master, but instead use devel as a master branch.

With this in mind, I'd like to request we stop doing this when creating new git repos, and use the default master branch that git defaults to. In our tooling, if we depend on a project that depends on devel (eg: ansible/ansible) we have the ability in zuul to define default-branch for said project.

tl;dr please stop renaming master to devel and using it as master branch.

thanks!
